pkg/util/sysutil/sysutil.go

```diff
@@ -70,3 +70,13 @@ func IsErrConnectionReset(err error) bool {
 func IsErrConnectionRefused(err error) bool {
 	return errors.Is(err, syscall.ECONNREFUSED)
 }
+
+// InterruptSelf sends SIGHUP to the process itself.
+func InterruptSelf() error {
+	pr, err := os.FindProcess(os.Getpid())
+	if err != nil {
+		// No-op.
```

Comment:
Does this noop comment also intend a return err here?

Reply:
Good call. Fixed.